packages/itwinui-react/src/core/AvatarGroup/AvatarGroup.test.tsx

```diff
@@ -157,11 +157,17 @@ it.each(['small', 'medium', 'large', 'x-large'] as Array<
     </AvatarGroup>,
   );
 
-  expect(
-    container.querySelectorAll(
-      `.iui-avatar-list > .iui-avatar[data-iui-size="${size}"]`,
-    ).length,
-  ).toBe(6);
+  if (size !== 'medium') {
+    expect(
+      container.querySelectorAll(
+        `.iui-avatar-list > .iui-avatar[data-iui-size="${size}"]`,
+      ).length,
+    ).toBe(6);
+  } else {
+    expect(
+      container.querySelectorAll(`.iui-avatar-list > .iui-avatar`).length,
+    ).toBe(6);
+  }
```

Comment:
nit: You can add the if-else/turnery check for the querySelector string instead of for the expect.

```jsx
expect(container.querySelectorAll( size !== medium ? "…" : "…" ))
```

Reply:
Done in this [commit.](https://github.com/iTwin/iTwinUI/pull/1935/commits/2a7e3590d20c4637003dc979ee24f0fde12f2314)